extern/sector-storage/selector_alloc.go

```diff
@@ -64,8 +64,50 @@ func (s *allocSelector) Ok(ctx context.Context, task sealtasks.TaskType, spt abi
 	return false, nil
 }
 
-func (s *allocSelector) Cmp(ctx context.Context, task sealtasks.TaskType, a, b *workerHandle) (bool, error) {
-	return a.utilization() < b.utilization(), nil
+func (s *allocSelector) Cmp(ctx context.Context, task sealtasks.TaskType, spt abi.RegisteredSealProof, a, b *workerHandle) (bool, error) {
+	if s.ptype == storiface.PathSealing {
```

Comment:
The only task type which isn't using `storiface.PathSealing` for the selector is FinalizeSector. I'm not sure if this will actually a noticeable difference

Reply:
When the sealing worker and storage worker are deployed separately, it is hoped that the data on the storage worker can be evenly distributed